docs/ABOUT.md

```diff
@@ -1,7 +1,9 @@
-Elm's goal is to be the best of functional programming in the browser.
+Elm is a pure, functional language that compiles to javascript.
 
-One of the primary goals of Elm is to completely eliminate runtime exceptions before they reach production by combining an aggressively helpful compiler with solid architectural principles.
+It was designed by Evan Czaplicki as part of writing his thesis on [Concurrent FRP for Functional GUIs](http://elm-lang.org/papers/concurrent-frp.pdf) and evolved into a language to help make making user interfaces better.
 
-Elm was designed by Evan Czaplicki as part of writing his thesis on [Concurrent FRP for Functional GUIs](http://elm-lang.org/papers/concurrent-frp.pdf).
+Elm's goal is to be the simplest and best language to use to make web-based user interfaces.
```

Comment:
I would suggest changing this to 

> Elm's goal is to be the simplest and best language for making web-based user interfaces.

to avoid repeating "to" twice in such close proximity.

Reply:
Good call!